llvm/lib/Target/AArch64/AArch64TargetTransformInfo.cpp

```diff
@@ -4923,9 +4923,7 @@ InstructionCost AArch64TTIImpl::getPartialReductionCost(
         return Invalid;
       break;
     case 16:
-      if (AccumEVT == MVT::i64)
-        Cost *= 2;
-      else if (AccumEVT != MVT::i32)
+      if (AccumEVT != MVT::i32)
```

Comment:
Why are you making this change?

Reply:
It's due to: https://github.com/llvm/llvm-project/pull/136173#discussion_r2053920360

Reply:
If we allow this case make sure to rename the test from "not_dotp" to "dotp".

Reply:
Good spot. Done.